stagemonitor-requestmonitor/src/test/java/org/stagemonitor/requestmonitor/tracing/B3Propagator.java

```diff
@@ -0,0 +1,21 @@
+package org.stagemonitor.requestmonitor.tracing;
+
+import io.opentracing.mock.MockSpan;
+import io.opentracing.mock.MockTracer;
+import io.opentracing.propagation.Format;
+import io.opentracing.propagation.TextMap;
+
+public class B3Propagator implements MockTracer.Propagator {
+	@Override
+	public <C> void inject(MockSpan.MockContext ctx, Format<C> format, C carrier) {
+		if (format instanceof B3HeaderFormat) {
+			((TextMap) carrier).put(B3HeaderFormat.SPAN_ID_NAME, Long.toHexString(ctx.spanId()));
```

Comment:
Don't know how to access them. However it is good enough for unit tests. This is only for the MockTracer afer all